test/Serialization/Poly.jl

```diff
@@ -0,0 +1,115 @@
+# Setup for fields
+R, x = PolynomialRing(QQ, "x")
+q = x^2 + 3//4
+K, a = NumberField(q)
+Ky, y = K["y"]
+Tow, b = NumberField(y^2 + 1, "b")
+NonSimRel, c = NumberField([y^2 - 5 * a, y^2 - 7 * a])
+Zt, t = PolynomialRing(ResidueRing(ZZ, 2), "t")
+Fin, d = FiniteField(t^2 + t + 1)
+Frac = FractionField(R)
+
+cases = [
+    [QQ, fmpq(3, 4), fmpq(1, 2)],
+    [ZZ, 3, 4],
+    [ResidueRing(ZZ, 6), 3, 5],
+    [K, a, a + 1],
+    [Tow, a^2 * b, a + b],
+    [NonSimRel, c[1], c[2] * a],
+    [Fin, d, 0],
+    [Frac, 1 // x, x^2]
+]
+
+function get_hom(R1::T, R2::T) where T <: Union{MPolyRing, PolyRing}
+    D = coefficient_ring(R1)
+    I = coefficient_ring(R2)
+
+    if D isa NfAbsNS
+        return hom(D, I, gens(I))
+
+    elseif D isa Hecke.NfRel{nf_elem}
```

Comment:
This is just the default case, right? So you could remove it?
```suggestion
    if D isa Hecke.NfRel{nf_elem}
```